tests/neg/import-implied.scala

```diff
@@ -0,0 +1,29 @@
+class TC
+object A {
+  implied tc for TC
+  def foo given TC = ()
+}
+object B {
+  import A._
+  foo            // error: no implicit argument was found
+  foo given tc   // error: not found: tc
```

Comment:
Forbidding explicit usage of `tc` here is a little counter-intuitive to lexical scoping, given the literal reading of `import A._`.

What about the following:

- `import A._` to import all members for _explicit_ usage
- `implied A._` to enable implied instances defined in `A` for _implicit_ resolution
- `implied A.c` make `A.c` available for implicit resolution, no matter _A.c_ is implicit or not.

The last one is up to debate, the idea is to encourage _use-site implicits_ over _definition-site implicits_.

Reply:
Would it be possible to use this feature to make any member a candidate for implicit resolution in scope, i.e. `import implied A.foo`?